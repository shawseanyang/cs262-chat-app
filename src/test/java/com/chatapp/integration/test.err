Feb 20, 2023 9:17:17 PM com.chatapp.server.ChatServer start
INFO: Server started, listening on 8000
Feb 20, 2023 9:18:08 PM com.chatapp.server.ChatServer$ChatServiceImpl$1 onNext
INFO: Created account for a
