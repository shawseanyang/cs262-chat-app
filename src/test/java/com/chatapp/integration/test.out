-> Error: Unknown command: a
-> Error: You must connect to a server first.
-> Error: You must connect to a server first.
